src/css/docs/components/table-wrap.css

```diff
@@ -0,0 +1,14 @@
+@media (hover: hover) {
+  .DocsMarkdown--table-wrap:hover {
+    width: calc(100% + var(--docs-body-sidebar-width));
+  }
+
+  .DocsMarkdown--table-wrap:hover table {
+    background-color: #fff;
```

Comment:
Shouldn't this be

```css
background-color: var(--background-color);
```

And then we can remove the `[theme="dark"]...` declaration?

Reply:
Thanks. I will push a fix accordingly.